src/client/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -96,7 +96,9 @@ public async Task<AuthenticationResult> RunAsync(CancellationToken cancellationT
 
                         authenticationResult.AuthenticationResultMetadata.DurationTotalInMs = sw.ElapsedMilliseconds;
                         authenticationResult.AuthenticationResultMetadata.DurationInHttpInMs = apiEvent.DurationInHttpInMs;
-                        authenticationResult.AuthenticationResultMetadata.DurationInCacheInMs = apiEvent.DurationInCacheInMs;                        
+                        authenticationResult.AuthenticationResultMetadata.DurationInCacheInMs = apiEvent.DurationInCacheInMs;
+
+                        authenticationResult.AuthenticationResultMetadata.AzureRegionAutoDetected = Region.RegionManager.AutoDiscoveredRegion;
```

Comment:
We seem to have two different names for this value. autoDetected and autoDiscovered. Is this intentional?

Reply:
They seem to be used interchangeably. Yea, we should be consistent. @neha-bhargava Do you know how it's mentioned in the specs, what's official term (auto-discover vs auto-detect)?

Reply:
This term is used interchangeably in the docs too. Good point to make it consistent. Since we use `TryAutoDetect` value to indicate that MSAL should discover the region we can use auto-detect to be consistent.